templates/products/new-product-single.php

```diff
@@ -252,67 +252,50 @@ class="vendor-earning simple-product"
 
                                     <?php do_action( 'dokan_product_edit_after_pricing', $post, $post_id ); ?>
 
-                                    <?php if ( dokan_get_option( 'product_category_style', 'dokan_selling', 'single' ) == 'single' ): ?>
-                                        <div class="dokan-form-group">
-                                            <label for="product_cat" class="form-label"><?php esc_html_e( 'Category', 'dokan-lite' ); ?></label>
-                                            <?php
-                                            $product_cat = -1;
-                                            $term = array();
-                                            $term = wp_get_post_terms( $post_id, 'product_cat', array( 'fields' => 'ids') );
-
-                                            if ( $term ) {
-                                                $product_cat = reset( $term );
+                                        <?php
```

Comment:
kindly move all the calculation related code to a function so that it can be reusable later on. Template file should be responsible for output only